threddsDev/notebooks/dataset_to_xarray_notebook.ipynb

```diff
@@ -0,0 +1,1108 @@
+      ],
+      "text/plain": [
+       "<xarray.DataArray 'LambertConformal_Projection' ()>\n",
+       "array(0, dtype=int32)\n",
+       "Coordinates:\n",
+       "    reftime  datetime64[ns] 2022-11-16T14:00:00\n",
+       "Attributes:\n",
+       "    grid_mapping_name:              lambert_conformal_conic\n",
+       "    latitude_of_projection_origin:  25.0\n",
+       "    longitude_of_central_meridian:  265.0\n",
+       "    standard_parallel:              25.0\n",
+       "    earth_radius:                   6371229.0\n",
+       "    _CoordinateTransformType:       Projection\n",
+       "    _CoordinateAxisTypes:           GeoX GeoY"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_da = ds[var_name.value]\n",
+    "new_da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Further xarray turtorial can be found [here](https://tutorial.xarray.dev/overview/xarray-in-45-min.html) and documentation is [here](https://docs.xarray.dev/en/stable/). It is a very powerful package for multi-index (x,y,z,time, etc.) datasets!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# your code here!"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3.9.13 ('ml_scratch')",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.13"
+  },
+  "viewer_info": {
+   "accepts": {},
```

Comment:
I'll take a look at what needs to go in here to target the right datasets

Reply:
Thanks!